transformation/standardJavascriptTransforms/src/metadataUpdater.js

```diff
@@ -46,6 +49,15 @@ function main(context) {
     );
   }
 
```

Comment:
Rather than make a whole new block, check if context is a map and then do `context.get("rules")` or do `context.rules` to reduce the duplicated code.